lux/core/frame.py

```diff
@@ -65,10 +65,10 @@ def __init__(self, *args, **kw):
         self._prev = None
         super(LuxDataFrame, self).__init__(*args, **kw)
 
-        self.executor_type = "Pandas"
-        self.executor = PandasExecutor()
-        self.SQLconnection = ""
-        self.table_name = ""
+        lux.config.executor_type = "Pandas"
+        lux.config.executor = PandasExecutor()
+        lux.config.SQLconnection = ""
```

Comment:
for `frame.py` could we just keep the `lux.config.SQLconnection` and `lux.config.table_name` unspecified and adopt their default values set in the config?

Reply:
Originally I set the parameters within config, but importing the PandasExecutor caused some circular dependency so I just set the values within frame.py

Reply:
SQLconnection's default value is now set in the config. The table_name parameter will remain a frame level parameter to accommodate use case where users work with multiple Lux dataframe each associated with a different database table.